### PR TITLE
feat(control_debug_tools): controller-dynamics consistency checker created.

### PR DESCRIPTION
**Pull request by HansOersted**
## Description
A mismatch of the parameters between the controllers and the dynamics may cause failing to stabilize the ego.
Therefore, this ticket creates a controller-dynamics consistency checker.

It checks the following parameters:

###### MPC-related parameters
input_delay
vehicle_model_steer_tau
acceleration_limit
steer_rate_lim_dps_list_by_curvature
steer_rate_lim_dps_list_by_velocity
###### PID related parameters:
max_acc
min_acc
accelerate_delay

## Related links
https://tier4.atlassian.net/jira/software/c/projects/RT1/boards/438?selectedIssue=RT1-5682

## Tests performed

image

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].
- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: control/control_debug_tools/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 3.5)
+project(control_debug_tools)
+
+find_package(autoware_cmake REQUIRED)
+autoware_package()
+
+find_package(builtin_interfaces REQUIRED)
+find_package(rosidl_default_generators REQUIRED)
+find_package(std_msgs REQUIRED)
```
Comment (takayuki5168) [inline control/control_debug_tools/CMakeLists.txt:9]: These seem not necessary. Please remove them, and check the build again.

File: control/control_debug_tools/package.xml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+  <depend>autoware_auto_planning_msgs</depend>
+  <depend>geometry_msgs</depend>
+  <depend>motion_utils</depend>
+  <depend>nav_msgs</depend>
+  <depend>rclcpp</depend>
+  <depend>rclcpp_components</depend>
+  <depend>tier4_autoware_utils</depend>
+  <depend>tier4_debug_msgs</depend>
+  <depend>tier4_planning_msgs</depend>
```
Comment (takayuki5168) [inline control/control_debug_tools/package.xml:27]: Remove dependency which you are not using

File: control/control_debug_tools/package.xml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  <maintainer email="takayuki.murooka@tier4.jp">Takayuki Murooka</maintainer>
+  <license>Apache License 2.0</license>
+
+  <author email="zhe.shen@tier4.jp">Zhe Shen</author>
+
+  <buildtool_depend>ament_cmake_auto</buildtool_depend>
+  <buildtool_depend>autoware_cmake</buildtool_depend>
+  <buildtool_depend>eigen3_cmake_module</buildtool_depend>
+
+  <build_depend>rosidl_default_generators</build_depend>
```
Comment (takayuki5168) [inline control/control_debug_tools/package.xml:17]: Remove

File: control/control_debug_tools/CMakeLists.txt (hunk 1 above)
Comment (HansOersted) [reply control/control_debug_tools/CMakeLists.txt]: removed in c30816d8a398a220f1a6c2db0a552f3875aebca8
built successfully!
image

File: control/control_debug_tools/package.xml (hunk 2 above)
Comment (HansOersted) [reply control/control_debug_tools/package.xml]: removed in c30816d8a398a220f1a6c2db0a552f3875aebca8

File: control/control_debug_tools/package.xml (hunk 3 above)
Comment (HansOersted) [reply control/control_debug_tools/package.xml]: removed in c30816d8a398a220f1a6c2db0a552f3875aebca8

**Review by takayuki5168 — APPROVED** [review]

**Pull request merged**